x-pack/test_serverless/functional/test_suites/security/config.context_awareness.ts

```diff
@@ -9,19 +9,15 @@ import { createTestConfig } from '../../config.base';
 
 export default createTestConfig({
   serverlessProject: 'security',
-  testFiles: [require.resolve('../common/discover/context_awareness')],
+  testFiles: [require.resolve('./ftr/one_discover/context_awareness')],
```

Comment:
Curious, what's the `one_discover` convention mean semantically?

Reply:
Actually, `One discover` is the name of the project that discover is going through it enables solutions to customize various `discover` elements.

I am happy to change it to `discover` if `one_discover` is confusing and need not be used in code. Will appreciate any comments from you as well @davismcphee

Reply:
Makes sense.. done here: https://github.com/elastic/kibana/pull/199818/commits/39a02c40af32359edc294cbe42e663a9bf4bb04b